### PR TITLE
[3.1] Add Support For Balance and Billing History

### PR DESCRIPTION
**Pull request by mrsimonbennett**

### CODE REVIEW

File: src/Entity/Balance.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ */
+final class Balance extends AbstractEntity
+{
+    /**
+     * @var string
+     */
+    public $monthToDateBalance;
+
+    /**
+     * @var string
```
Comment (GrahamCampbell) [inline src/Entity/Balance.php:27]: is this a string? what's an example value?

File: src/Entity/BillingHistory.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+     */
+    public $description;
+
+    /**
+     * @var string
+     */
+    public $amount;
+
+    /**
+     * @var string
```
Comment (GrahamCampbell) [inline src/Entity/BillingHistory.php:32]: is this a string?

File: src/Entity/BillingHistory.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+ */
+final class BillingHistory extends AbstractEntity
+{
+    /**
+     * @var string
+     */
+    public $description;
+
+    /**
+     * @var string
```
Comment (GrahamCampbell) [inline src/Entity/BillingHistory.php:27]: is this a string?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
Thanks for the PR. I've left a few inline comments. Please could you update the readme with the new stuff too?

File: src/Entity/Balance.php (hunk 1 above)
Comment (mrsimonbennett) [reply src/Entity/Balance.php:27]: I based off the digitalocean api definition of attribute with money been a string.

This is inline with the rest of the project too 

https://github.com/DigitalOceanPHP/Client/blob/d000d351d931902d4bb26e2836a3bf52cba28c86/src/Entity/Size.php#L55

**Comment by mrsimonbennett** [comment]
Closed and adding more endpoints as required so its not going to be merge ready

**Pull request closed without merging**